packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -80,6 +80,11 @@ export class VaButton {
    */
   @Prop({ reflect: true }) uswds?: boolean = true;
 
+  /**
+   * If `false`, buttons of type submit will run onsubmit call back, but not submit form.
+   */
+  @Prop() dontSendForm?: boolean = false;
```

Comment:
Instead of creating a new prop, is it possible to leverage the existing `submit` prop to determine the different ways we need or want it to submit?

Reply:
Changed the `submit` to a string,  'submit' or any truthy value will run the form's callback (`form.onsubmit`) and submit the data;  'prevent' will  run call back but not send data (essentially a faux `preventDefault`), 'skip' will  run call back but send data, but skip the form's callback.